### DISCUSSION TITLE
FSUIPC Offsets

### DISCUSSION THREAD

**Discussion by AGA058** [flybywiresim/aircraft#3146] [Help]
Hello Guys,

i was wondering if there is a possibility to interface with the mod and FSUIPC. I know that there is a list with FUSIPC7 but i think the A320 has more functions for example every single letter on the MCDU. 
Is there a Offset list for the A32NX or do you know another method to interface with selfbuild hardware and the Sim?

Thank you all!

Best regards,
Marvin

**Comment by riveroalbert** [comment]
Thats interesting. FSUIPC7 is still in beta so i guess it will evolve. 
For example i guess there must be an offset for managed/selected mode... (might just be my ignorance, but I think heading_slot_index_set is non editable)

**Reply by frogmonkey2** [reply]
from what I understand it's now available for MSFS2020 and should allow for quite a few more commands than you see in the defaults. I am purchasing to set up slot commands on my keyboard for push/pull commands for the altitudes/speed/heading modes and other commands to make flying easier that chasing around knobs with a mouse

**Comment by aguther** [comment]
This might help you:
https://docs.flybywiresim.com/start/autopilot-fbw/#mapping-of-events-to-control-autopilot-autothrust

**Reply by frogmonkey2** [reply]
thanks for this
